tests/qos/qos_sai_base.py

```diff
@@ -3,19 +3,25 @@
 import pytest
 import re
 import yaml
+import random
+import os
+import sys
 
-from tests.common.fixtures.ptfhost_utils import ptf_portmap_file    # lgtm[py/unused-import]
+from tests.common.fixtures.ptfhost_utils import ptf_portmap_file  # noqa F401
 from tests.common.helpers.assertions import pytest_assert, pytest_require
 from tests.common.mellanox_data import is_mellanox_device as isMellanoxDevice
-from tests.common.dualtor.dual_tor_utils import upper_tor_host,lower_tor_host,dualtor_ports
-from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports, get_mux_status, check_mux_status, validate_check_result
-from tests.common.dualtor.constants import UPPER_TOR, LOWER_TOR
+#from tests.common.dualtor.dual_tor_utils import upper_tor_host, lower_tor_host, dualtor_ports  # noqa F401
```

Comment:
@congh-nvidia Can you elaborate why comment out this import line even when the fixtures imported here are used by this script?

Reply:
Hi @wangxin, I think it must be a typo when I was fixing the pre-commit issues. This line should not be comment out. I think, it didn't break the test because the fixtures are also imported in the pytest plugin. Did you find it when converting the test to Python3, if so please remove the comment; if not, I'll fix this. Thanks!

Reply:
Hi @congh-nvidia, I noticed this when I was doing cherry-pick. Not in python3 migration. It would be great if you can submit a PR to fix this. Thanks!

Reply:
Sure, I'll fix it, thanks!

Reply:
@wangxin The PR is opened: https://github.com/sonic-net/sonic-mgmt/pull/7965. Thanks.

Reply:
Thank you @congh-nvidia !